iGCS/UserInterface/ViewControllers/DefaultSettingsTableViewController.m

```diff
@@ -0,0 +1,234 @@
+@property (nonatomic, retain) NSArray *otherSettingsArray;
+
+@property (nonatomic, retain) NSMutableArray *sectionKeysArray;
+@property (nonatomic, retain) NSMutableDictionary *sectionContentsDict;
+
+@property (strong, nonatomic) UIBarButtonItem *editBarButtonItem;
+@property (strong, nonatomic) UIBarButtonItem *doneBarButtonItem;
+
+@property (nonatomic, assign) BOOL standardSelected;
+@end
+
+@implementation DefaultSettingsTableViewController
+
+- (instancetype)initWithStyle:(UITableViewStyle)style {
+    self = [super initWithStyle:style];
+    if (self) {
+        // Custom initialization
+    }
+    return self;
+}
+
+- (void) viewDidLoad {
+    [super viewDidLoad];
+    [self configureNavigationBar];
+    [self createSectionData];
+}
+
+- (void) createSectionData {
+    
+    //general settings
+    self.generalSettingsArray = @[@"Contact Us", @"Order iDroneLink"];
+    
+    //waypoint settings
+    //Remove For this Release
+    //self.waypointSettingsArray = @[@"Altitude", @"Radius"];
+    
+    //other settings
+    self.otherSettingsArray = @[@"Radio",@"About"];
+    
+    NSMutableArray *keys = [[NSMutableArray alloc] init];
+    NSMutableDictionary *contents = [[NSMutableDictionary alloc] init];
+    
+    NSString *generalSectionKey = @"General";
+    //Remove for this Release
+    //NSString *waypointSectionKey = @"Waypoints";
+    NSString *otherSectionKey = @"Other";
+    contents[generalSectionKey] = self.generalSettingsArray;
+    //Remove for this Release
+    //contents[waypointSectionKey] = self.waypointSettingsArray;
+    contents[otherSectionKey] = self.otherSettingsArray;
+
+    
+    [keys addObject:generalSectionKey];
+    //Remove for this Release
+    //[keys addObject:waypointSectionKey];
+    [keys addObject:otherSectionKey];
+    
+    [self setSectionKeysArray:keys];
+    [self setSectionContentsDict:contents];
+}
+
+-(void)configureNavigationBar {
+    [self setTitle:@"Settings"];
+    
+    self.doneBarButtonItem = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemDone
+                                                                             target:self action:@selector(doneWithChanges:)];
+    
+    self.navigationItem.rightBarButtonItem = self.doneBarButtonItem;
+}
+
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return [[self sectionKeysArray] count];
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    NSString *key = [self.sectionKeysArray objectAtIndex:section];
+    return [self.sectionContentsDict[key] count];
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    static NSString *cellIdentifier = @"Cell";
+    NSString *key = [self.sectionKeysArray objectAtIndex:indexPath.section];
+    NSArray *contents = [self.sectionContentsDict objectForKey:key];
```

Comment:
Use the new style to get objects out of a dictionary: `self.sectionContentsDict[key]`